src/main/scala/org/broadinstitute/dsde/firecloud/service/DataUseRestrictionSupport.scala

```diff
@@ -141,6 +164,30 @@ trait DataUseRestrictionSupport extends LazyLogging {
     result
   }
 
+
+  // Takes in any Product (case classes extend Product) and a prefix and creates a map of all the fields and their values
+  // with the prefix prepended to the field
+//  private def formatWithPrefix(prefix: String, product: Product): Map[String, JsValue] = {
+//    implicit object AnyJsonFormat extends RootJsonFormat[Any] {
+//      def write(x: Any) = x match {
+//        case n: Int => JsNumber(n)
+//        case s: String => JsString(s)
+//        case b: Boolean if b == true => JsTrue
+//        case b: Boolean if b == false => JsFalse
+//        case a: Array[Any] => a.
+//      }
+//      def read(value: JsValue) = value match {
+//        case JsNumber(n) => n.intValue()
+//        case JsString(s) => s
+//        case JsTrue => true
+//        case JsFalse => false
+//      }
+//    }
+//
+//    val values = product.productIterator
+//    product.getClass.getDeclaredFields.map( prefix + _.getName -> values.next.toJson ).toMap
+//  }
+
```

Comment:
This is what I tried to do to generalize adding a prefix to the field names of an object. 

The problem is values.next is of type `Any`. It's all fun and games until your value isn't atomic. So if I have an Any that might be an Array[Any], I'm unsure how to define the json write function to handle that. 

Ideas and suggestions are welcome